pkg/component/etcd/monitoring.go

```diff
@@ -30,6 +30,7 @@ import (
 const (
 	monitoringPrometheusJobEtcdNamePrefix          = "kube-etcd3"
 	monitoringPrometheusJobBackupRestoreNamePrefix = "kube-etcd3-backup-restore"
+	monitoringPrometheusJobCompactionNamePrefix    = "kube-etcd3-compaction"
```

Comment:
```suggestion
	monitoringPrometheusJobCompactionNamePrefix    = "etcd-druid"
```

This should be named `etcd-druid` because these metrics are exposed from druid, and should not specify anything about which exact metrics are being exposed by druid. Because druid can expose any metrics in the future, and not restricted to only compaction-related metrics.